test/common.go

```diff
@@ -20,10 +22,14 @@ var arrayUsername = flag.String("arrayUsername", "xxx", "Array Username")
 
 var arrayPassword = flag.String("arrayPassword", "xxx", "Array Password")
 
+var downstreamArrayIP = flag.String("downstream", "", "Array IP to be used as downstream")
```

Comment:
Please explain what a 'downstream' is.

Reply:
Replication direction from Upstream Array to Downstream Array.

Reply:
I meant to explain it in the option description.  For example: "IP address of an array to be used as a downstream replica in tests that configure replication".

If this option is not always required, please specify when it must be provided.  Same for the other options.